source/docs/3/digitizing_basics.rst

```diff
@@ -40,287 +40,240 @@ Data Source [LINZ]_
-    is useful when you want to make adjustments after the feature is created.
-    You can also delete a selected vertex by clicking the :kbd:`Delete` key.
-    (:kbd:`Option+Delete` on a mac)
-
-.. image:: /static/3/digitizing_basics/images/19.png
-   :align: center
-
-20. Once you have finished digitizing all the roads, click the
-    :guilabel:`Toggle Editing` button. Click :guilabel:`Save`.
-
-.. image:: /static/3/digitizing_basics/images/20.png
-   :align: center
-
-21. In the :guilabel:`New GeoPackage Layer` dialog, click the :guilabel:`...`
-    button and select GeoPackage database named ``digitizing.gpkg``. Name the new layer as an attribute called ``Parks``. and select ``MultiPolygon`` as the :guilabel:`Type`. The base topographic map is in the ``EPSG:2193 - NZGD 2000`` CRS.  Click :guilabel:`OK`. In :guilabel:`New Field` Enter ``Name`` of the type :guilabel:`Text data`, with ``50`` as :guilabel:`Maximum length` and click :guilabel:` Add to attribute list.`
+ .. image:: /static/3/digitizing_basics/images/13.png
+    :align: center
+   
+14. The default style of the new line layer is a thin line. Let's change it to better see the digitized features on the canvas. Select the ``Roads`` layer and click :guilabel:`Layer Styling Panel`.
 
+ .. image:: /static/3/digitizing_basics/images/14.png
+    :align: center   
+ 
+15. In the :guilabel:`Layer Styling Panel`, search for different `road` layer styles. Select ``topo road``. Click :guilabel:`OK`.
+
+ .. image:: /static/3/digitizing_basics/images/15.png
+    :align: center
+   
+16. Now the road layer will be clearly visible. If you are satisfied with the work, click :guilabel:`Save Layer Edits` button to save the changes. 
+
+ .. image:: /static/3/digitizing_basics/images/16.png
+    :align: center
+	
+17. Before we digitize the remaining roads, it is essential to update some other important snap settings to create an error-free layer. Right-click on any space on the toolbar area and activate the :guilabel:`Snapping toolbar.` 
+
+ .. image:: /static/3/digitizing_basics/images/17.png
+    :align: center
+   
+18. Now an :guilabel:`Enable Snapping` (Magnet Icon) will appear on the panel. Click on it to enable it and select :guilabel:`All Layers` and choose ``Open Snapping Options..``. 
+
+ .. image:: /static/3/digitizing_basics/images/18.png
+    :align: center 
+	
+19. In the :guilabel:`Snapping options` dialog, click the :guilabel:`Snapping on Intersection`, which allows you to snap on an intersection of a background layer.
+
+ .. image:: /static/3/digitizing_basics/images/19.png
+    :align: center
+   
+20. Now you can click :guilabel:`Add feature` button and digitize other roads around the park. Make sure to click :guilabel:`Save Edits` after adding a new feature to save your work. A helpful tool to help you digitize is the **Vertex Tool**. Click the :guilabel:`Vertex Tool` button and select ``Vertex Tool (Current Layer)``.

```

Comment:
Highlight Save Edits button in screenshot

Reply:
Done.